data-migration/migrate-sharding-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,439 @@
+```
+
+In the command above:
+
+- `-u` or `--user` specifies the user name to be used. If a password is required for authentication, you can use `-p $YOUR_SECRET_PASSWORD` to pass the password to Dumpling.
+- `-p` or `--password` specifies the password to be used.
+- `-p` or `--port` specifies the port to be used.
+- `-h` or `--host` specifies the IP address of the data source.
+- `-t` or `--thread` specifies the number of threads for the export. Increasing the number of threads improves the concurrency of Dumpling and the export speed, and increases the database's memory consumption. Therefore, it is not recommended to set the number too large.
+- `-o` or `--output` specifies the export directory of the storage, which supports a local file path or a [URL of an external storage](/br/backup-and-restore-storages.md).
+- `-r` or `--row` specifies the maximum number of rows in a single file. If you use this parameter, Dumpling enables the in-table concurrency to speed up the export and reduce the memory usage.
+- `-F` specifies the maximum size of a single file. The unit is `MiB`. Inputs such as `5GiB` or `8KB` are also acceptable. It is recommended to keep the value to 256 MiB or less, if you use TiDB Lightning to load this file into a TiDB instance.
+- `-B` or `--database` specifies databases to be exported.
+- `-f` or `--filter` export tables that match the filter pattern. For the filter syntax, see [table-filter](/table-filter.md).
+
+> **Note:**
+>
+> If the size of a single exported table exceeds 10 GiB, it is strongly recommended to use the `-r` and `-F` options.
+
+Then, run the following command to use Dumpling to export `table3` and `table4` from `my_db2`:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup dumpling -h ${ip} -P 3306 -u root -t 16 -r 200000 -F 256MB -B my_db2 -f 'my_db2.table[34]' -o ${data-path}/my_db2
+```
+
+The reason for storing all source data tables in one directory is to facilitate subsequent migration by TiDB Lightning.
+
+The starting point information needed for incremental replication is in the `${data-path}` directory, and in the `metadata` files of `my_db1` and `my_db2`. They are meta-information files automatically generated by Dumpling. To perform incremental replication, you need to note down the binlog location information.
+
+Congratulations! Now you have exported all the required full data.
+
+## Step 2. Start TiDB Lightning to migrate data
+
+Before starting TiDB Lightning for migration, it is recommended that you understand how to handle checkpoints, and then choose the appropriate way to proceed according to your needs.
+
+### Checkpoints
+
+Migrating a large volume of data usually takes hours or even days. If such a long running processes unfortunately crashes, it can be very frustrating to redo everything from scratch.
+
+Fortunately, TiDB Lightning provides a `checkpoints` feature to let you store the migration progress, so that migration task can resume from the breakpoint upon restart.
```

Comment:
```suggestion
Migrating a large volume of data usually takes hours or even days. There is a certain chance that the long-running process is interrupted and restarts again. It can be very frustrating to redo everything from scratch.

Fortunately, TiDB Lightning provides a `checkpoints` feature to let you store the migration progress, so that an interrupted migration task can resume from the breakpoint upon restart.
```